docs/gitbook/architecture/public-private-control-model.md

```diff
@@ -0,0 +1,50 @@
+
+## How the PPC model routes updates to private and shared chains
+
+The PPC model and the *Interbit Middleware* permit applications to dispatch actions to private and shared chains. Specifically, *Chain Sponsorship* is used to manage the ACL updates and other configuration required to allow users to successfully dispatch actions to a permissioned blockchain while ensuring that private keys never leave the user's device.
+
+Once configured, *Interbit Middleware* mediates routes actions dispatched from the application to the appropriate chain using a friendly chain alias. It also subscribes to chain updates and updates the application's redux store.
+
+![Interacting with chains in the PPC Model](PPC-Dispatch-Actions.png)
+[View the diagram at *sequencediagram.org*](http://sequencediagram.org/index.html#initialData=C4S2BsFMAIBEQM4AcCGwDGALEA7A5tCuqAPY4LTAnQAKATiAG5owo4Am00AypinZE4BhPrgQAoVHVDoQqHMGgAiALIBPAIJIk0XAB0cAITokA7gkh0lhCsbMW6k-jLltFSgEqCArgA8eVALWKBRe7H5O0iCy8u4AkgqWAEZgBiog7OxQpvyQwRTpmdm5kS6xylpI4NFoIGQG9EwsBiIouPnQlY2MpdGuChXa1ei19Ti8uewtojgdlRMC7L0xboNVNaQ4DQzMwJDTbbM2ndrd4ngm3joAqg6ExHXk0CPeCLgESDsszzPQV+wscTIIiQcREUC7GB2cyWMEPSHQMIRcFNPbQQpZSA5ARwiHfLo7cRJEj+EiMSwnJDdABc0FulgA5BQaN4ksNoABpSBqAwoTKCSjUDRCAAy0FMmEgW3ISDICBIDHwP0ORJJ0DJFOhDlp9Lo9026uln1RMABwBQ4i1lgAtAA+DHFAS0+DAjCYfWPcQOrG5O0EpjOxCoN3K3Aesiq0nkvX+xi01ph-4sJbEqMU2O0wzgEjoADW4lj1rt3uxkFpWEOADoEKyEOgGElIAAKACUXoymNLdqRvkDrqwfyQZv2OBRZEF0AQgVBPaLtqtdB1cUHw4kwPQoPYkBRCNj4i3O++JZKB-h3x7++3Z7RC-EUqWFxIVzpdzHTxebyVCD4i1DOBXgLrqCh43iYMKOCBMAXpB6Ido6wHXjAe4wfMP6CJG6rRpS3QADSoZMtLCmKbCcAA6gwaIAFIkPoODoGQABmIB4N4v4SlKBjIHKCrvH+GEanqC46q+DzjmM36TNAZoWguxZwT6TpwEGaADm+7ZFApkB+qcOx9sGA4VmGamFra+GLLSXgALYkGi7DKW6BhUJOaGcIZOD8VhZmCPGvxJnsKZqgJlILN50BZjm+Zeewc7HopbnVrW9YgI2rbqZ2vq2j2ekqe6fkjm+E5TgqM4+L4c5CXSy55WuqAbpeKE6Uw9WIcFLnNXiaKxZuV4dVBpXtSaYVgQ4d4cEAA)
+
+## Chain Authorization (cAuth) in the PPC Model
+
+The PPC model and the template are designed to enable Chain Authorization (cAuth). Chain Authorization gives the user fine grained control over how and where their private data is shared and used with other Interbit applications.
```

Comment:
This is an optional addition to PPC and we should specify that. It is not strictly part of PPC